src/main/java/gift/config/AuthInterceptor.java

```diff
@@ -19,6 +19,9 @@ public AuthInterceptor(UserService userService) {
 
     @Override
     public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        if ("OPTIONS".equalsIgnoreCase(request.getMethod())) { // preFlight 해결
```

Comment:
주석 달아주신 부분까지 좋은 것 같습니다! 💯  요렇게 변경되면 더욱 좋을 것 같아요~
```suggestion
        if(HttpMethod.valueOf(request.getMethod()).equals(HttpMethod.OPTIONS))
```